### PR TITLE
Fixes wiki books and default URLs

### PR DESCRIPTION
**Pull request by ike709**

### CODE REVIEW

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -78,10 +78,10 @@ GLOBAL_VAR(bot_ip)
 
 	var/server
 	var/banappeals
-	var/wikiurl = "http://www.tgstation13.org/wiki" // Default wiki link.
-	var/forumurl = "http://tgstation13.org/phpBB/index.php" //default forums
-	var/rulesurl = "http://www.tgstation13.org/wiki/Rules" // default rules
-	var/githuburl = "https://www.github.com/tgstation/-tg-station" //default github
+	var/wikiurl = "https://ftl13.com/wiki/index.php?title=" // Default wiki link.
```
Comment (ike709) [inline code/controllers/configuration.dm:81]: This is fine, it redirects to main page

File: code/game/objects/items/weapons/manuals.dm [hunk 2]
```diff
@@ -772,7 +772,7 @@
     			}
 			</script>
 			<p id='loading'>You start skimming through the manual...</p>
-			<iframe width='100%' height='97%' onload="pageloaded(this)" src="[config.wikiurl]/[page_link]?printable=yes&remove_links=1" frameborder="0" id="main_frame"></iframe>
```
Comment (ike709) [inline code/game/objects/items/weapons/manuals.dm:775]: This bit never worked correctly anyways

File: config/config.txt [hunk 3]
```diff
@@ -162,7 +162,7 @@ CHECK_RANDOMIZER
 # FORUMURL http://ftl13.com/phpBB/index.php
 
 ## Wiki address
-# WIKIURL http://www.ftl13.com/wiki
+# WIKIURL https://ftl13.com/wiki/index.php?title=
```
Comment (ike709) [inline config/config.txt:165]: Make sure the config is updated to be like this

**Comment by ike709** [comment]
I don't know why the FTL drive fix is included.

**Comment by ninjanomnom** [comment]
because you're doing everything in the equivalent of your master branch, I said to branch off of your rebase branch lol

**Comment by ike709** [comment]
I did do that damn it

**Pull request merged**